### PR TITLE
Add backup container for recurring dropbox backup

### LINKED ISSUE #1: Backup Script
Add a periodic backup script that makes both local and online backups just like the one on the [wiki](https://github.com/metakgp/metakgp-wiki/blob/master/jobs/backup/backup_to_dropbox.py).

### PR DESCRIPTION
**Pull request by shikharish**
Closes #1

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -2,4 +2,10 @@ DATABASE_HOST=localhost
 DATABASE_PORT=5432
 DATABASE_NAME=databaseofbabel
 DATABASE_USERNAME=metakgp_user
-DATABASE_PASSWORD=somerandomstringfordevelopment
\ No newline at end of file
+DATABASE_PASSWORD=somerandomstringfordevelopment
+PGPASSFILE=
```
Comment (harshkhandeparkar) [inline .env.template]: Can you add some documentation for these env vars?

**Review by harshkhandeparkar — COMMENTED** [review]
Any plans to add a rotate backup script?

**Review by harshkhandeparkar — COMMENTED** [review]
Have you tested this?

**Comment by shikharish** [comment]
Yes

**Review by harshkhandeparkar — APPROVED** [review]
Then I'll approve it but waiting for @rajivharlalka's approval before merging.

**Comment by proffapt** [comment]
@rajivharlalka look at this once, thanks.

File: backup/backup_to_dropbox.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            print("Uploaded: ", float(uploaded) / (1024 * 1024), "MB")
+            while next_chunk:
+                client.files_upload_session_append_v2(next_chunk, cursor)
+                uploaded += len(next_chunk)
+                cursor = dropbox.files.UploadSessionCursor(session.session_id, uploaded)
+                print("Uploaded: ", float(uploaded) / (1024 * 1024), "MB")
+
+                next_chunk = f.read(chunksize)
+
+            commit_info = dropbox.files.CommitInfo(path="/" + file_name)
```
Comment (rajivharlalka) [inline backup/backup_to_dropbox.py:47]: assuming this works correctly.

File: backup/run_backup.sh [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+if ! $rotate_backup; then
+	echo "failed to rotate backups!"
+	# Notify Slack
+	if [[ -n "$SLACK_INCIDENTS_WH_URL" ]]; then
+		curl -s -H 'content-type: application/json' \
+			-d "{ \"text\": \"🔴DoB Dropbox backup rotation failure on ${timestamp}\" }" \
+			"$SLACK_INCIDENTS_WH_URL"
+	fi
+	exit 1
+fi
```
Comment (rajivharlalka) [inline backup/run_backup.sh]: I think this works for us right now, but shouldn't rotation should come before uploading the backup ? whatif the new backup has very high size and fails, i guess we should take this in consideration. Though shouldn't be an issue now.

File: docker-compose.yml [hunk 4]
```diff
@@ -1,24 +1,46 @@
...
-            - postgres-of-babel
-      environment:
-        - POSTGRES_USER=${DATABASE_USERNAME}
-        - POSTGRES_PASSWORD=${DATABASE_PASSWORD}
-        - POSTGRES_DB=${DATABASE_NAME}
-        - PGPORT=${DATABASE_PORT}
-        - PGHOST=${DATABASE_HOST}
-      volumes:
-        - db-volume:/var/lib/postgresql/data
+    image: postgres
```
Comment (rajivharlalka) [inline docker-compose.yml]: tag version of postgres being used.

**Review by rajivharlalka — APPROVED** [review]

File: docker-compose.yml (hunk 4 above)
Comment (shikharish) [reply docker-compose.yml]: we are using the latest version. shouldnt be a problem in the future i think.

File: backup/run_backup.sh (hunk 3 above)
Comment (shikharish) [reply backup/run_backup.sh]: makes sense

File: backup/backup_to_dropbox.py (hunk 2 above)
Comment (shikharish) [reply backup/backup_to_dropbox.py:47]: it does :)

**Review by harshkhandeparkar — APPROVED** [review]
I had approved this. If @rajivharlalka approves, it can be merged.

**Review by rajivharlalka — APPROVED** [review]

**Pull request merged**